tutorials/create-action-email/create-action-email.md

```diff
@@ -0,0 +1,67 @@
+---
+author_name: Christian Savchev
+author_profile: https://github.com/christiansavchev
+keywords: tutorial
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product-function>sap-btp-cockpit ]
+primary_tag: software-product>sap-btp--cloud-foundry-environment
+parser: v2
+---
+
+# Create an Action of a Type Email
+<!-- description --> Learn how to create an action of a type email.
+
+## Prerequisites
+- You have activated Alert Notification in a trial account as described in [Set Up SAP Alert Notification service for SAP BTP for Initial Use](set-up-alert-notification)
+
+## You will learn
+- How to create an action of a type email, so that you can receive notifications to a given email
+    > If you want to receive notifications to other delivery channels, see the SAP Cloud Platform Alert Notification documentation at [Managing Actions](https://help.sap.com/docs/ALERT_NOTIFICATION/5967a369d4b74f7a9c2b91f5df8e6ab6/8a7e092eebc74b3ea01d506265e8c8f8.html).
```

Comment:
I believe this note is causing the text on row 19 to be separated from the bullet point.  I recommend that you move this note to the intro section below.